sdk/keyvault/azure-security-keyvault-certificates/test/ut/certificate_client_base_test.hpp

```diff
@@ -41,130 +40,47 @@ namespace Azure {
      */
     CertificateContentType ContentType;
   };
-  /**
-   * @brief Client Certificate Credential authenticates with the Azure services using a
-   * Tenant ID, Client ID and a client secret.
-   *
-   */
-  class TestClientSecretCredential final : public Core::Credentials::TokenCredential {
-  public:
-    Core::Credentials::AccessToken GetToken(
-        Core::Credentials::TokenRequestContext const& tokenRequestContext,
-        Core::Context const& context) const override
-    {
-      Core::Credentials::AccessToken accessToken;
-      accessToken.Token = "magicToken";
-      accessToken.ExpiresOn = DateTime::max();
-
-      if (context.IsCancelled() || tokenRequestContext.Scopes.size() == 0)
-      {
-        accessToken.ExpiresOn = DateTime::min();
-      }
-
-      return accessToken;
-    }
-  };
 
   class KeyVaultCertificateClientTest : public Azure::Core::Test::TestBase,
                                         public ::testing::WithParamInterface<int> {
+
   private:
     std::unique_ptr<Azure::Security::KeyVault::Certificates::CertificateClient> m_client;
-    std::string GetEnv(const std::string& name, std::string const& defaultValue = std::string())
-    {
-#if defined(_MSC_VER)
-#pragma warning(push)
-#pragma warning(disable : 4996)
-      const char* ret = std::getenv(name.data());
-#pragma warning(pop)
-#else
-      const char* ret = std::getenv(name.data());
-#endif
-
-      if (!ret)
-      {
-        if (!defaultValue.empty())
-        {
-          return defaultValue;
-        }
-
-        throw std::runtime_error(
-            name + " is required to run the tests but not set as an environment variable.");
-      }
-
-      return std::string(ret);
-    }
 
   protected:
-    std::shared_ptr<Azure::Identity::ClientSecretCredential> m_credential;
-    std::shared_ptr<TestClientSecretCredential> m_testCredential;
+    std::shared_ptr<Core::Credentials::TokenCredential> m_credential;
     std::string m_keyVaultUrl;
-    std::chrono::milliseconds m_defaultWait;
+    std::chrono::milliseconds m_defaultWait = 20s;
```

Comment:
This is a declaration, so you must've been doing `using namespace` somewhere in the header, or it is done by the other header we include. Avoid it.